### PR TITLE
Drop Python 3.8/Add Python 3.13

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -6,6 +6,8 @@ Unreleased
 ==========
 * ``THUMBNAIL_STORAGE`` should now be an alias in the Django ``STORAGES`` setting.
   The old way of specifying a dotted path to a Storage module is still supported.
+* Confirmed support for Python 3.13.
```
Comment (dulmandakh) [inline CHANGES.rst:9]: Maybe add a note that Python 3.13 is supported on Django 5.1 only

**Comment by dulmandakh** [comment]
LGTM 👍

File: CHANGES.rst (hunk 1 above)
Comment (claudep) [reply CHANGES.rst]: OK, done.

**Review by dulmandakh — APPROVED** [review]

**Comment by claudep** [comment]
Thanks!

**Pull request merged**